### DISCUSSION TITLE
Termux in pkgs.org and/or repology.org ?

### DISCUSSION THREAD

**Discussion by maaaaz** [termux/termux-packages#14277] [Ideas]
Hello dear Termux team,

Would it be a good idea for Termux to ask [pkgs.org](https://pkgs.org/) and/or [repology.org](https://repology.org/) to be listed as a proper "distribution" ?

Being integrated on these metadata platforms would help us, simple users, to find information in a centralized place !

Cheers !

**Comment by sylirre** [comment]
Termux is already on Repology: https://repology.org/repository/termux. Here is where we store package metadata: https://github.com/termux/repology-metadata/blob/master/packages.json

Feel free to ask pkgs.org to add the Termux.
